.github/workflows/update-docs.yml

```diff
@@ -1,18 +1,13 @@
 name: Update MeshLib Documentation site
 
 on:
-  release:
-    types: [ published ]
-  # Allows you to run this workflow manually from the Actions tab
-  workflow_dispatch:
```

Comment:
why remove this, how would we update on release?

Reply:
reworked